dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/CompanyUploader.kt

```diff
@@ -126,4 +127,33 @@ class CompanyUploader(
             )
         }
     }
+
+    /**
+     * Updates the ISINs of all companies.
+     * @param leiIsinMap the delta-map with the format "LEI"->"ISIN1,ISIN2,..."
+     */
+    fun updateIsinMapping(leiIsinMap: Map<String, String>) {
+        retryOnCommonApiErrors {
+            for ((lei, isins) in leiIsinMap) {
+                val isinList = isins.split(",").map { it.trim() }
+                val company = companyDataControllerApi.getCompaniesBySearchString(lei)
```

Comment:
I hate that we use this endpoint for this type of search. We should have a proper `GET /companies/identifiers/{identiferType}/identifier` endpoint. But nothing to do here.